### ISSUE TITLE
Resonite bot online status

### ISSUE THREAD

**Issue by XDelta** [Yellow-Dog-Man/Resonite-Issues#3443]
### Is your feature request related to a problem? Please describe.

I'd like the Resonite bot to always appear as online. Currently it will show as offline so it ends up shuffled into the other offline users, often leading to need to search for it to send commands. Also leads to some occasional confusion in discord when new players see it and it appears offline.

### Describe the solution you'd like

The Resonite bot to always appear online (atleast when network/chat services are available) or otherwise be pinned to the top/available in some other quick manner.

### Describe alternatives you've considered

Search for it each time

### Additional Context

https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/41

### Requesters

_deltawolf

**Comment by shiftyscales** [comment]
This is effectively a duplicate of #62 / #41 which are dependent on #1116.

Time could potentially be put towards this- but it still doesn't resolve the underlying issue- as the account would still be mixed among other users in your contacts list- just slightly further up it. If the account were set sociable instead of online- it could be slightly further up still.

I don't think this is really worth putting much engineering effort into directly, however.

**Comment by carson-coder** [comment]
(Sorry if this is unrelated) It would be nice to just be able to pin accounts on your contacts list

**Comment by shiftyscales** [comment]
That would be related to the aforementioned issues above, @carson-coder. If you wish to show support / interest in a specific issue, the best way to do so would be to thumb up the original post of whatever issues you wish to show support of.

https://github.com/Yellow-Dog-Man/Resonite-Issues/blob/main/HOW_WE_PRIORITIZE.md
